packages/sdks/src/functions/evaluate/evaluate.ts

```diff
@@ -54,12 +16,22 @@ export function evaluate({
   rootSetState,
   event,
   isExpression = true,
-  enableCache,
```

Comment:
not mention in changelogs that we removed all eval caching code, because we already disabled it in a previous PR a while back